### PR TITLE
Add goci lint.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go [hunk 1]
```diff
@@ -118,8 +118,8 @@ var (
 const (
 	clusterManagerFinalizer = "operator.open-cluster-management.io/cluster-manager-cleanup"
 	clusterManagerApplied   = "Applied"
-	clusterManagerAvailable = "Available"
-	caBundleConfigmap       = "ca-bundle-configmap"
+	// clusterManagerAvailable = "Available" // not used
```
Comment (qiujian16) [inline pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go]: remove if not used

File: pkg/helpers/queuekey_test.go [hunk 2]
```diff
@@ -204,7 +204,7 @@ func TestClusterManagerDeploymentQueueKeyFunc(t *testing.T) {
 			fakeOperatorClient := fakeoperatorclient.NewSimpleClientset(c.clusterManager)
 			operatorInformers := operatorinformers.NewSharedInformerFactory(fakeOperatorClient, 5*time.Minute)
 			store := operatorInformers.Operator().V1().ClusterManagers().Informer().GetStore()
-			store.Add(c.clusterManager)
+			_ = store.Add(c.clusterManager)
```
Comment (qiujian16) [inline pkg/helpers/queuekey_test.go]: I think it is better to add error check here like

```go
if err := store.Add(c.clusterManager); err != nil {
    t.Fatal(err)
}
```

File: pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller.go [hunk 3]
```diff
@@ -43,11 +43,12 @@ type klusterletLocker struct {
 }
 
 const (
-	bootstrapSecret         = "BootstrapSecret"
-	bootstrapSecretDegraded = "BootstrapSecretDegraded"
-	hubConfigSecret         = "HubConfigSecret"
-	hubConfigSecretDegraded = "HubConfigSecretDegraded"
-	hubConnectionDegraded   = "HubConnectionDegraded"
+	// TODO: The following SSARs are not used yet.
```
Comment (qiujian16) [inline pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller.go]: remove these if not used

File: pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller_test.go [hunk 4]
```diff
@@ -121,7 +121,7 @@ func TestSync(t *testing.T) {
 			return
 		}
 		ssar := &authorizationv1.SelfSubjectAccessReview{}
-		json.Unmarshal(data, ssar)
+		_ = json.Unmarshal(data, ssar)
```
Comment (qiujian16) [inline pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller_test.go]: add error check

**Review by qiujian16 — COMMENTED** [review]
I think we should always add error check rather than ignoring the err

**Comment by xuezhaojun** [comment]
I guess a lot of errors are ignore, I will fix those soon.

File: pkg/helpers/queuekey_test.go (hunk 2 above)
Comment (xuezhaojun) [reply pkg/helpers/queuekey_test.go]: Solved.

File: pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go (hunk 1 above)
Comment (xuezhaojun) [reply pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go]: Removed.

File: pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller.go (hunk 3 above)
Comment (xuezhaojun) [reply pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller.go]: Removed.

File: pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller_test.go (hunk 4 above)
Comment (xuezhaojun) [reply pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller_test.go]: Added.

File: test/integration/integration_suite_test.go [hunk 5]
```diff
@@ -38,7 +38,7 @@ const (
 	eventuallyTimeout  = 30 // seconds
 	eventuallyInterval = 1  // seconds
 	hubNamespace       = "open-cluster-management-hub"
-	spokeNamespace     = "open-cluster-management-agent"
+	// spokeNamespace     = "open-cluster-management-agent"
```
Comment (qiujian16) [inline test/integration/integration_suite_test.go]: remove this line if we do not need this

Comment (xuezhaojun) [reply test/integration/integration_suite_test.go]: Removed.

**Pull request merged**